Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.6-alpine3.8 as BUILD
+
+RUN apk add git postgresql-dev gcc musl-dev libffi-dev make gnupg
+
+COPY setup.py /root/LocalEGA/setup.py
+COPY lega /root/LocalEGA/lega
+COPY requirements.txt /root/LocalEGA/requirements.txt
+
+RUN pip install -r /root/LocalEGA/requirements.txt && \
+    pip install /root/LocalEGA
+
+FROM python:3.6-alpine3.8
```

Comment:
Not already on line 1 ?

Reply:
Nope, the final image are built from this line and on.